toolkit/scripts/toolkit.mk

```diff
@@ -30,7 +32,7 @@ clean-package-toolkit:
 	rm -f $(toolkit_remove_archive)
 	rm -rf $(toolkit_build_dir)
 
-package-toolkit: go-tools
+package-toolkit: go-tools $(toolkit_rpms_snapshot)
```

Comment:
Not sure we want to be super pedantic here with the dependencies, but take a look at `$(local_specs)` in srpm_pack.mk. It adds a dependency on every spec file itself. The directory dependency will track adding/removing folders but not changes to files inside those folders.

Reply:
Oh, we (I) want to be as pedantic as humanly possible, so thank you for pointing this out. I've updated the targets quite a bit since this comment to make take into account all possible dependencies. I've only skipped `DIST_TAG` since this one shouldn't be manually modified.